android/src/main/java/com/reactcommunity/rndatetimepicker/RNDatePickerDialogModule.java

```diff
@@ -171,6 +171,9 @@ private Bundle createFragmentArguments(ReadableMap options) {
     if (options.hasKey(RNConstants.ARG_NEUTRAL_BUTTON_LABEL) && !options.isNull(RNConstants.ARG_NEUTRAL_BUTTON_LABEL)) {
       args.putString(RNConstants.ARG_NEUTRAL_BUTTON_LABEL, options.getString(RNConstants.ARG_NEUTRAL_BUTTON_LABEL));
     }
+    if (options.hasKey(RNConstants.ARG_TZOFFSET_MIN) && !options.isNull(RNConstants.ARG_TZOFFSET_MIN)) {
```

Comment:
I'd perhaps suggest to rename `ARG_TZOFFSET_MIN` to `ARG_TZOFFSET_MINS` because `MIN` gives a feeling of being some sort of a "minimum", whereas reading `MINS` will be understood as "minutes"